classes/class-wpcom-liveblog-rest-api.php

```diff
@@ -289,10 +355,12 @@ public static function crud_entry( WP_REST_Request $request ) {
 
 		// Get the required parameters from the request
 		$crud_action = $request->get_param( 'crud_action' );
+		$json 	 	 = $request->get_json_params();
+
 		$args = array(
-			'post_id'  => $request->get_param( 'post_id' ),
-			'content'  => $request->get_param( 'content' ),
-			'entry_id' => $request->get_param( 'entry_id' ),
+			'post_id'  => self::get_json_param( 'post_id', $json ),
```

Comment:
It is passed is as the second parameter. `self::get_json_param()` acts as a helper function to check whether the first parameter key is set and return it, otherwise it will return false.